### DISCUSSION TITLE
Small screen customizations

### DISCUSSION THREAD

**Discussion by Annunaki** [agronick/aa-torque#112] [Q&A] [answered]
My vehicle has a screen that is just 8 inch, is it possible to reduce the size of the gauges/overall app size somehow?

Currently AA-Torque is taking up all of the screen in AA and I cant seem to find a way to have it share the screen real estate with maps as shown in the introduction video on YouTube.

At the moment the screen is split with Spotify and Maps but when selecting AA-Torque, it opens up on the whole of the screen, Maybe there is a setting somewhere that I have overlooked?

Cheers

**Comment by agronick** [comment, marked as answer]
See https://github.com/agronick/aa-torque/issues/85#issuecomment-1854037413

**Reply by Annunaki** [reply]
Thanks, I didn't check in issues before posting as didn't think it was an issue but more of a customization thing.

Thanks all the same.
